### PR TITLE
FT-82 # Create the plugin info command

### PR DESCRIPTION
**Pull request by simonmarklar**
# Added
`plugin info` command

Notes: FT-91 commits will be rebased out once they have been merged

### CODE REVIEW

**Comment by mymattcarroll** [comment]
I figured i would just a once over of all the docs. Just noticed on the main `README.md` it says:

> Currently only works with Blink Forms definitions.

Is that still true?

File: docs/plugin-development-overview.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+The plug-in can optionally define a build step to build the source files into a bundled module.
+
+# Plugin API
+
+A Forms CLI plug-in must expose an object of the following structure
+
+```javascript
+{
+  build: (cfg) => Promise,
```
Comment (mymattcarroll) [inline docs/plugin-development-overview.md]: Do these promises need to resolve to anything special?

File: packages/forms-template-helper/docs/sample-plugin-entrypoint.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Sample plugin index.js
+
+See [/docs/sample-plugin-index.md](https://github.com/blinkmobile/forms-cli/tree/master/docs/sample-plugin-index.md) for an example of how the forms template helper can be used in a plugin for the forms cli.
```
Comment (mymattcarroll) [inline packages/forms-template-helper/docs/sample-plugin-entrypoint.md]: Will this link work once the PR is merged?

File: docs/sample-plugin-index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Sample plugin index.js
+
+Below is the entrypoint for the [AngularJS Plugin](packages/bm-plugin-forms-angularjs)
+
+
+```
```
Comment (mymattcarroll) [inline docs/sample-plugin-index.md]: if you put `js` here, i think youll get some pretty colours and stuff

File: docs/plugin-development-overview.md (hunk 1 above)
Comment (simonmarklar) [inline docs/plugin-development-overview.md]: ah dang it, No, the promises dont, but processForm does not return a promise, its an array of functions. i've got that documented elsewhere but mis-documented it here

File: packages/forms-template-helper/docs/sample-plugin-entrypoint.md (hunk 2 above)
Comment (simonmarklar) [inline packages/forms-template-helper/docs/sample-plugin-entrypoint.md]: i put relative links in, looks like github auto expanded them!

**Comment by mymattcarroll** [comment]
LGTM 👍

**Pull request merged**